sig-governance.md

```diff
@@ -103,6 +103,7 @@ The process for closing a SIG/WG is as follows:
   - Move all appropriate github repositories to an appropriate archive or a repo outside of the Kubernetes org
     - Each subproject a SIG owns must transfer ownership to a new SIG, outside the project, or be retired
     - File an issue with kubernetes/org if multiple repos that need to be retired
+    - Move the existing SIG directory into the [archive](/archive) in kubernetes/community
```

Comment:
This shouldn't be a subpoint of the github repo one. It falls under `Update `sigs.yaml` to reflect the removal of the SIG/WG`

Reply:
Done.